plugins/UsersManager/API.php

```diff
@@ -264,6 +267,64 @@ private function getDefaultUserPreference($preferenceName, $login)
         }
     }
 
+    /**
+     * Returns all users with their role for $idSite.
+     *
+     * @param int $idSite
+     * @param int|null $limit
+     * @param int|null $offset
+     * @param string|null $filter_search text to search for in the user's login, email and alias (if any)
+     * @param string|null $filter_access only select users with this access to $idSite. can be 'noaccess', 'some', 'view', 'admin', 'superuser'
+     *                                   Filtering by 'superuser' is only allowed for other superusers.
+     * @return array
+     */
+    public function getUsersPlusRole($idSite, $limit = null, $offset = 0, $filter_search = null, $filter_access = null)
+    {
+        if (!$this->isUserHasAdminAccessTo($idSite)) {
```

Comment:
Maybe we can require at least view access when calling this method and maybe not "anonymous"?

Reply:
If the user only has view access to the site, should they be able to see & search through what other users have access to it? Actually, right now the users admin page only allows admins to view it, so changing this would only affect API users.

Reply:
> If the user only has view access to the site, should they be able to see & search through what other users have access to it?

View users should not be able to see any other user or user information. @diosmosis this is important, maybe you saw some code giving view users some access to view some other users info?